Make mashlib work as a library again

Vinnl opened the pull request:
Instead of initialising the Data Browser when the DOM content has
loaded, it now adds an initialiser function to `window.UI`. This
can then be called by the file including it.

Thus, like before, the Data Browser's HTML actually initiates the initialisation of the Data Browser. However, I couldn't put back the exact same code as earlier, since bf5c67523a046097b5cbbcfbde47e88103b52faf introduced header and footer initialisation code, which references other mashlib functions. Thus, I chose to simply expose the initialisation on the already-existing `UI` global. This adds yet another global API, so suggestions for better approaches are welcome.

Fixes #58.

timbl commented:
It is a bug to distribute the databrowser.html file as a file called index.html, as it will confuse people and servers wherever it is.

Note that the databrowser.html restored in https://github.com/solid/mashlib/pull/61  has the line
`const panes = require('mashlib')`
where panes gives you solid-panes and panes.UI gives you solid-ui
That is the old API.
IMHO we should support that API for a while.  Not just delete it

timbl commented:
So in summary, support globals $rdf, UI, panes  long term, and the require('mashlib') kludge for a year or so.

Vinnl commented:
@timbl I've incorporated your comments, please review again.

One thing to note is that I did not remove the generated `index.html` yet, because NSS [has been updated](https://github.com/solid/node-solid-server/commit/e364266629cee5606070778cda25abd7664528e7#diff-7356ba1cd5bf2e897c784cad5f7b59a7R92) to look for that instead of `databrowser.html`. I added a comment mentioning that, so it can be removed when NSS has been updated after this release.
